<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetaliedPostView</class>
 <widget class="QWidget" name="DetaliedPostView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>955</height>
   </rect>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1A1A1B;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QScrollArea" name="MainWindowScrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background: transparent;
</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QWidget" name="mainWindowScrollAreaWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1424</width>
        <height>915</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="MainWindowLayout" stretch="0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="HeaderBar">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetFixedSize</enum>
         </property>
         <property name="leftMargin">
          <number>35</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="logoLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:white;
background-color: rgba(255, 255, 255, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/images/logo.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
border-radius: 21px;
background-color: rgb(85, 85, 85);
color:white;
margin-left:60px;
margin-right:10px;
border:none;
padding:15px;
</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Search...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profileButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 0);
border-radius: 6px;
border:none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/images/UserSettings.png</normaloff>C:/Users/images/UserSettings.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>43</width>
             <height>43</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameSpacingHeader">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notificationButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 0);
border-radius: 6px;
border:none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/images/ButtonNotificationImage.png</normaloff>C:/Users/images/ButtonNotificationImage.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="shortcutSettingsButton">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 0);
border-radius: 6px;
border:none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/images/ProfileSettingsShortcut.png</normaloff>C:/Users/images/ProfileSettingsShortcut.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>10</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="HeaderDelimiter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(116, 116, 116);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="MainLayout" stretch="0,0,1,0,0">
         <property name="rightMargin">
          <number>40</number>
         </property>
         <property name="bottomMargin">
          <number>40</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="accesLinksLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetFixedSize</enum>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="homeBtn">
               <property name="styleSheet">
                <string notr="true">background:none;
background-color: rgba(255, 255, 255, 0);
font: 700 14pt &quot;Raleway&quot;;
color: rgb(192, 192, 192);
margin-top:6px;
text-align: left;
padding-left:30px;</string>
               </property>
               <property name="text">
                <string>Home</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="recommendedBtn">
               <property name="styleSheet">
                <string notr="true">background:none;
background-color: rgba(255, 255, 255, 0);
font: 700 14pt &quot;Raleway&quot;;
color: rgb(192, 192, 192);
text-align: left;
padding-left:30px;</string>
               </property>
               <property name="text">
                <string>Recommended</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="communitiesBtn">
               <property name="styleSheet">
                <string notr="true">background:none;
background-color: rgba(255, 255, 255, 0);
font: 700 14pt &quot;Raleway&quot;;
color: rgb(192, 192, 192);
text-align: left;
padding-left:30px;</string>
               </property>
               <property name="text">
                <string>Communities</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="communitiesWidgetLayout">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SizeConstraint::SetFixedSize</enum>
               </property>
               <property name="leftMargin">
                <number>28</number>
               </property>
               <property name="topMargin">
                <number>6</number>
               </property>
               <property name="rightMargin">
                <number>12</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="myProfileBtn">
               <property name="styleSheet">
                <string notr="true">background:none;
background-color: rgba(255, 255, 255, 0);
font: 700 14pt &quot;Raleway&quot;;
color: rgb(192, 192, 192);
text-align: left;
padding-left:30px;</string>
               </property>
               <property name="text">
                <string>My Profile</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="settingsBtn">
               <property name="styleSheet">
                <string notr="true">background:none;
background-color: rgba(255, 255, 255, 0);
font: 700 14pt &quot;Raleway&quot;;
color: rgb(192, 192, 192);
text-align: left;
padding-left:30px;</string>
               </property>
               <property name="text">
                <string>Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="disconnectBtn">
               <property name="styleSheet">
                <string notr="true">background:none;
background-color: rgba(255, 255, 255, 0);
font: 700 14pt &quot;Raleway&quot;;
color: rgb(192, 192, 192);
margin-bottom:14px;
text-align: left;
padding-left:30px;</string>
               </property>
               <property name="text">
                <string>Disconnect</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="HeaderDelimiter_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>1</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>1</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(116, 116, 116);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="FriendsLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetFixedSize</enum>
             </property>
             <property name="bottomMargin">
              <number>30</number>
             </property>
             <item>
              <widget class="QFrame" name="friendsWidhtDelimiter_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>245</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="titleFriendsLayout">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>9</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="friendsTitleLabel">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 700 12pt &quot;Raleway&quot;;
color: rgb(160, 160, 160);
margin-left:13px;</string>
                  </property>
                  <property name="text">
                   <string>Friends</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="searchBarFriends">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border:none;
background-color:#3A3A3A;
padding-left:6px;
color: rgb(255, 255, 255);
font: 10pt &quot;Raleway&quot;;
border-radius:10px;</string>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="frame">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>Search</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_myFeed_5">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 700 12pt &quot;Raleway&quot;;
color: rgb(160, 160, 160);
margin-left:13px;</string>
               </property>
               <property name="text">
                <string>Online</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_myFeed_6">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 700 12pt &quot;Raleway&quot;;
color: rgb(160, 160, 160);
margin-left:13px;</string>
               </property>
               <property name="text">
                <string>Offline</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="onlineFriendsLayout">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
               </property>
               <property name="leftMargin">
                <number>9</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="offlineFriendsLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>40</number>
               </property>
              </layout>
             </item>
             <item>
              <widget class="QFrame" name="frame_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>1</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="friendsPostsDelimiteer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>2</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(116, 116, 116);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="postAreea">
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="postAreaRecommendPostsDelimiter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>2</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(116, 116, 116);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>15</number>
           </property>
           <property name="rightMargin">
            <number>15</number>
           </property>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
